server/src/test/java/io/crate/execution/dml/upsert/TransportShardUpsertActionTest.java

```diff
@@ -327,38 +323,4 @@ public void test_dynamic_insert_of_integer_upcasted_to_long_can_be_replicated()
         // verifies that it does not throw a ClassCastException: class java.lang.Integer cannot be cast to class java.lang.Long
         transportShardUpsertAction.processRequestItemsOnReplica(indexShard, request);
     }
-
-    /**
-     * This tests verifies a regression introduced in 5.3 where a marker was introduced in order to skip items instead
-     * of relying on a NULL source. On request coming from older nodes this marker isn't available and the source NULL
-     * check wasn't working as expected, resulted in an NPE.
-     */
-    @Test
```

Comment:
We can remove this test because we're not supporting these older formats at all any more.